release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -351,12 +351,12 @@ In the table below, features are marked with the following statuses:
 |Scheduler profiles
 |-
 |TP
-|
+|TP
 
 |Non-preempting priority classes
 |-
 |TP
-|
+|TP
```

Comment:
@ingvagabund Can you confirm that non-preempting priority classes are staying tech preview for 4.8?

Reply:
I don't recall there was any change in this respect. @soltysh the same question here as well. Is there a top level card which tracks which features is planned to be GA'ed?